src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -935,19 +960,27 @@ private string InvokeReadLine(CancellationToken cancellationToken)
             }
         }
 
+        // TODO: Should we actually be directly invoking input versus queueing it as a task like everything else?
```

Comment:
@SeeminglyScience IDK should we? It's kinda weird we just give up on the queue right here. Though we _are_ in between processing the queue so it...works.

Reply:
That's exactly what it does. It's annoyingly different!